apps/epic-web/src/inngest/functions/stripe/slack-monthly-reporter.ts

```diff
@@ -2,16 +2,14 @@ import {postToSlack} from '@skillrecordings/skill-api'
 import {WebClient} from '@slack/web-api'
 import {inngest} from 'inngest/inngest.server'
 import {
-  fetchCharges,
-  fetchRefunds,
-  SimplifiedCharge,
-  SimplifiedRefund,
+  fetchCombinedBalanceTransactions,
+  CombinedBalanceTransaction,
 } from 'lib/transactions'
 import {prisma} from '@skillrecordings/database'
 import {calculateTotals} from 'components/calculations/calculate-totals'
 import {calculateSplits} from 'components/calculations/calculate-splits'
 
-const ANNOUNCE_CHANNEL = 'C07RDAMQ7PG'
+const ANNOUNCE_CHANNEL = 'C03QFFWHT7D'
```

Comment:
we will get the whole monthly report in `lc-epic-web` for us - it's exactly the same as our complicated spread sheet (before expenses)